numpy/lib/tests/test_histograms.py

```diff
@@ -270,7 +270,7 @@ def test_object_array_of_0d(self):
             histogram, [np.array(0.4) for i in range(10)] + [np.inf])
 
         # these should not crash
-        np.histogram([np.array(0.5) for i in range(10)] + [.500000000000001])
+        np.histogram([np.array(0.5) for i in range(10)] + [.500000000000002])
```

Comment:
Note: This test was failing because one of the created bins had zero width. I assume this was not the intention of the test. It was added in https://github.com/numpy/numpy/pull/10268, which is about coercing values of object arrays. I've increased the value minimally to not run into the zero-width bin case.